packages/gitbook/src/app/(space)/layout.tsx

```diff
@@ -98,6 +98,7 @@ export default async function SpaceRootLayout(props: { children: React.ReactNode
                             headerTheme.backgroundColor.light,
                         )}
                         ${generateColorVariable('header-link', headerTheme.linkColor.light)}
+                        ${generateColorVariable('header-button-text', colorContrast(headerTheme.linkColor.light as string, ['#000', '#fff']))}
```

Comment:
Not for this PR: I'm just wondering if we are not starting to generate too many css variables, and how it impacts the HTML payload. Maybe if it's too big we should move this to a dynamic route to generate a `style.css`.

Reply:
I think it's OK right now, also Next.js becomes SPA once loaded so I guess it's fine.